nad

```diff
@@ -920,7 +928,14 @@ function setup_reverse() {
       } else {
         console.log("configuration error for reverse: no check found (hostname mismatch?)");
       }
-  });
+  }
+
+  circapi.setup(auth_token, 'nad', api_options);
+  if (check_bundle_id == null) {
+      circapi.get('/check_bundle?f_type=json:nad&f_target=' + hostname, null, circapi_cb);
+  else {
+      circapi.get('/check_bundle/' + check_bundle_id, null, circapi_cb);
```

Comment:
is this really a check_bundle and not a check?  Each check inside a check_bundle would have a different mtev:// reverse string.. we only want "ours."  If this actually works, it may not be an issue.